Dockerfile

```diff
@@ -4,21 +4,24 @@ COPY ./composer.json /tmp/src1/composer.json
 COPY ./composer.lock /tmp/src1/composer.lock
 
 WORKDIR /tmp/src1
-RUN composer install --no-dev --no-progress --no-autoloader
+RUN composer install --no-dev --no-progress --no-autoloader --ignore-platform-req=ext-pcntl
 
 COPY ./ /tmp/src2
 RUN cp -r /tmp/src1/vendor /tmp/src2/vendor
 WORKDIR /tmp/src2
-RUN composer install --no-dev --no-progress --optimize-autoloader
+RUN composer install --no-dev --no-progress --optimize-autoloader --ignore-platform-req=ext-pcntl
```

Comment:
I'm not entirely sure why this Dockerfile installs dependencies (twice?) in a dedicated image before copying dependencies over (not having to install composer in the production image?).

We could get rid of the extra flag if we installed it in the next step, but maybe it's ok as it is.